### PR TITLE
feat!: add `@taiga-ui/styles`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nsbarsukov** [comment]
Let's dont merge until merging of this PR
https://github.com/Tinkoff/taiga-ui/pull/2322

File: projects/addon-preview/README.md
Comment (waterplea) [inline projects/addon-preview/README.md:20]: It's not a base package, instead mention this package where there's a section about global styles

File: projects/core/components/group/group-styles.component.ts
Comment (waterplea) [inline projects/core/components/group/group-styles.component.ts:5]: This should be moved into component and out of the global styles.

File: package.json
Comment (waterplea) [inline package.json:134]: Why do we need to manually introduce webpack to deps?

File: projects/demo/src/modules/app/home/examples/main.md
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main.md:2]: Same here

File: projects/demo/src/modules/app/stackblitz/stackblitz-deps.constants.ts
Comment (waterplea) [inline projects/demo/src/modules/app/stackblitz/stackblitz-deps.constants.ts:27]: I believe these could be * now

File: projects/demo/src/modules/markup/skeleton/skeleton.template.html
Comment (waterplea) [inline projects/demo/src/modules/markup/skeleton/skeleton.template.html:55]: That is, if you added the styles package. We need to make sure to put some warning in all places like that. Also maybe it makes sense to add a special page about global styles so people will see its structure and maybe only import the parts they need.

File: projects/taiga-schematics/schematics/ng-add/tests/ng-add.spec.ts
Comment (waterplea) [inline projects/taiga-schematics/schematics/ng-add/tests/ng-add.spec.ts:113]: We should probably prompt people if they want to install global styles.

**Review by waterplea — COMMENTED** [review]
- We can probably remove all the -old files
- Need to make sure keyframes are not used inside required code
- normalize seems like external dep that was just copied over, maybe it should be made a proper dep or dropped

File: package.json
Comment (splincode) [reply package.json]: fixed

File: projects/addon-preview/README.md
Comment (splincode) [reply projects/addon-preview/README.md]: fixed

File: projects/demo/src/modules/app/home/examples/main.md
Comment (splincode) [reply projects/demo/src/modules/app/home/examples/main.md]: fixed

File: projects/demo/src/modules/app/stackblitz/stackblitz-deps.constants.ts
Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/stackblitz-deps.constants.ts]: fixed

File: projects/taiga-schematics/schematics/ng-add/tests/ng-add.spec.ts
Comment (splincode) [reply projects/taiga-schematics/schematics/ng-add/tests/ng-add.spec.ts:115]: fixed

File: projects/demo/src/modules/app/home/home.template.html
Comment (nsbarsukov) [inline projects/demo/src/modules/app/home/home.template.html:114]: Screenshot 2022-08-12 at 10 23 30

File: projects/demo/src/modules/app/home/home.template.html
Comment (nsbarsukov) [inline projects/demo/src/modules/app/home/home.template.html:107]: duplicates

File: projects/demo/src/modules/app/stackblitz/project-files/stackblitz.less.md
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/stackblitz.less.md:9]: why did we change file format ?

File: projects/styles/taiga-ui-global.less
Comment (nsbarsukov) [inline projects/styles/taiga-ui-global.less:1]: Some style-files (for example, `tui-form.less`) below uses constants from this file.

Can we really delete it ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/cdk/schematics/ng-add/schema.json
Comment (waterplea) [inline projects/cdk/schematics/ng-add/schema.json:19]: `global Taiga UI classes, such as 'tui-space', 'tui-skeleton', etc?` so that people know what it's about

File: projects/cdk/schematics/ng-update/steps/replace-styles.ts
Comment (waterplea) [inline projects/cdk/schematics/ng-update/steps/replace-styles.ts:12]: Fonts has to be in core because it's a part of the theme.

File: projects/demo/src/modules/app/home/examples/angular-json-styles.md
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/angular-json-styles.md:8]: Missing fonts, once you return them to core styles.

File: projects/demo/src/modules/app/home/home.template.html
Comment (splincode) [reply projects/demo/src/modules/app/home/home.template.html]: fixed

File: projects/demo/src/modules/app/stackblitz/project-files/stackblitz.less.md
Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/project-files/stackblitz.less.md]: Because this file was needed only for automatic generation of stackblitz examples

File: projects/cdk/schematics/ng-add/schema.json
Comment (splincode) [reply projects/cdk/schematics/ng-add/schema.json]: fixed

File: projects/cdk/schematics/ng-update/steps/replace-styles.ts
Comment (splincode) [reply projects/cdk/schematics/ng-update/steps/replace-styles.ts]: fixed

File: projects/demo/src/modules/app/home/examples/angular-json-styles.md
Comment (splincode) [reply projects/demo/src/modules/app/home/examples/angular-json-styles.md]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**
